javascript/javascript3/week3/TASK3-Shopping cart/index.js

```diff
@@ -1,130 +1,149 @@
-const totalPrice = document.getElementById("total-price");
-
-class Currency {
-  constructor(name, rate) {
-    this.name = name;
-    this.rate = rate;
-  }
-}
-
-const USD = new Currency("USD", 7.5);
-const EUR = new Currency("EUR", 8.2);
-const HUF = new Currency("HUF", 46.2);
+(function () {
+  const defaultCurrency = "DKK";
+  const totalPrice = document.getElementById("total-price");
+
+  class Product {
+    constructor(name, price) {
+      this.name = name;
+      this.price = price;
+    }
 
-class Product {
-  constructor(name, price) {
-    this.name = name;
-    this.price = price;
+    async convertToCurrency(currency) {
+      await getCurrencyRate(this.price, currency);
```

Comment:
You await, but you don't use the result. You can just say:
```
const result = getCurrencyRate(this.price, currency);
return result
```
or even better
`_return getCurrencyRate(this.price,currency)_ `